src/linttest/basic_test.go

```diff
@@ -23,6 +23,72 @@ func TestCallStaticParent(t *testing.T) {
 	runFilterMatch(test, "callStatic")
 }
 
+func TestVoidResultUsedInAssignment(t *testing.T) {
+	test := linttest.NewSuite(t)
+	test.AddFile(`<?php
+	/**
+	* @return void
+	*/
+	function f() {}
+	$_ = f();
+`)
+	test.Expect = []string{
+		`void function result used`,
+	}
+	test.RunAndMatch()
+}
+
+func TestVoidResultUsedInBinary(t *testing.T) {
+	test := linttest.NewSuite(t)
+	test.AddFile(`<?php
+	/**
+	* @return void
+	*/
+	function f() {}
+
+	f() & 1;
+	f() | 1;
+	f() ^ 1;
+	//f() && true; // skipped due to "undefined: Use true instead of true" report
```

Comment:
Related to #250.
For now, tests should declare builtin constants explicitly, like:
```
function define($_, $_) {}
define('false', 1 == 0);
define('true', 1 == 1);
```

Reply:
Should be fixed.